docs/protocols/amiga.md

```
@@ -608,32 +608,32 @@ Amiga keyboards handle CAPS LOCK with unique behavior requiring sophisticated sy
 - Press+release cycle toggles the state
 - Host sends LED state back to device
 
-**Synchronization Challenge**:
+**Synchronisation Challenge**:
 
-The converter must keep keyboard LED state and USB HID state synchronized, especially after reboot:
+The converter must keep keyboard LED state and USB HID state synchronised, especially after reboot:
 
 | Keyboard LED | USB HID | Converter Action | Reason |
```

markdownlint-cli2 (0.21.0)

[warning] Table pipe style
Expected: no_leading_or_trailing; Actual: leading_and_trailing; Unexpected leading pipe

(MD055, table-pipe-style)